examples/simple_repeater/MyMesh.cpp

```diff
@@ -787,7 +793,7 @@ void MyMesh::updateFloodAdvertTimer() {
 }
 
 void MyMesh::dumpLogFile() {
-#if defined(RP2040_PLATFORM)
+#if defined(RP2040_PLATFORM) || defined(ARCH_PORTDUINO)
```

Comment:
nit: can we name this `PORTDUINO_PLATFORM` to be more in line with the other defines?